test/protocol/ProtocolDiamondTest.js

```diff
@@ -1021,7 +1021,7 @@ describe("ProtocolDiamond", async function () {
         ).to.revertedWith(RevertReasons.INIT_ZERO_ADDRESS_NON_EMPTY_CALLDATA);
       });
 
-      it("should revert if _calldata is empty but _init is not address(0)", async () => {
+      it("Should revert if _calldata is empty but _init is not address(0)", async () => {
```

Comment:
"_calldata is empty but _init is not address(0)"

Reply:
done